atomic_reactor/plugins/pre_resolve_remote_source.py

```diff
@@ -179,14 +180,7 @@ def source_request_to_json(self, source_request):
     def get_koji_user(self):
         unknown_user = self.workflow.conf.cachito.get('unknown_user', 'unknown_user')
         try:
-            metadata = get_build_json()['metadata']
-        except KeyError:
-            msg = 'Unable to get koji user: No build metadata'
-            self.log.warning(msg)
-            return unknown_user
-
-        try:
-            koji_task_id = int(metadata.get('labels').get('koji-task-id'))
+            koji_task_id = int(self.workflow.user_params.get('koji-task-id'))
```

Comment:
https://github.com/containerbuildsystem/osbs-client/blob/master/osbs/build/user_params.py#L96

I'm very afraid that this must be `koji_task_id`